addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -137,6 +137,63 @@ MockServer.include({
             grouped_activities: groupedActivities,
         };
     },
+    /**
+     * Simulate the 'get_mention_suggestions' on 'res.partner'.
+     *
+     * @private
+     * @returns {Array[]}
+     */
+    _mockResPartnerGetMentionSuggestions(args) {
+        const search = args.kwargs.search || '';
+        const limit = args.kwargs.limit || 8;
+        const mentionSuggestionsFilter = function (partners, search, limit) {
```

Comment:
just `function mentionSuggestionsFilter(...)`, space before/after definition, and docstring

Reply:
My linter does not agree with scoped functions.

Reply:
Certainly because of scoped functions being globally scoped (same issue with `var`). If so, okay for `const`, but at least space before + docstring, so that this looks like an inner-function (and we can just skip reading it if naming is good enough)

Reply:
done